src/posit/connect/content.py

```diff
@@ -140,16 +294,15 @@ def restart(self) -> None:
                 f"Restart not supported for this application mode: {self.app_mode}. Did you need to use the 'render()' method instead? Note that some application modes do not support 'render()' or 'restart()'.",
             )
 
-    @overload
     def update(
         self,
         *,
         # Required argument
-        name: str,
+        name: Optional[str] = None,
         # Content Metadata
         title: Optional[str] = None,
         description: Optional[str] = None,
-        access_type: Literal["all", "acl", "logged_in"] = "acl",
+        access_type: Optional[Literal["all", "acl", "logged_in"]] = None,
```

Comment:
Same as above

Reply:
The difficult part is that the default value represents an actual value. 

When updating, only the values being set should be updated. If there is a default value, then that value should be used unless overwritten.

It was never an issue because the implementation previously did not _actually_ use the default value... It was just an `@override`

Reply:
When using `._Attrs` unpacking, there are no defaults... skirting the issue.

Closing